core/src/main/java/org/apache/carbondata/core/indexstore/AbstractTableDataMap.java

```diff
@@ -84,11 +121,11 @@
    * @param filterExp
    * @return
    */
-  boolean isFiltersSupported(FilterResolverIntf filterExp);
+  public abstract boolean isFiltersSupported(FilterResolverIntf filterExp);
```

Comment:
I feel it is better to accept a `FilterType` as input and return boolean, so client is not dependent on carbon's filter interface